templates/haproxy-reload

```diff
@@ -23,5 +23,7 @@ if [ -f "$HAPROXY_RELOAD_FILE" -o "$1" = "-f" ]; then
         # Exclude editor backup files
         flock -w 1 {{ LOAD_BALANCER_BACKENDS_DIR }} cat {{ LOAD_BALANCER_BACKENDS_DIR }}/*[^~]
     } > {{ LOAD_BALANCER_BACKEND_MAP }}
+    # Dump server state before restart
+    echo "show servers state" | socat /run/haproxy/admin.sock - > {{ LOAD_BALANCER_STATE_FILE }}
```

Comment:
`socat` needs to be added to [`LOAD_BALANCER_APT_PACKAGES`](https://github.com/open-craft/ansible-load-balancer/blob/23df7bc11bbcb27626ee1e98b048e15a65d0227b/defaults/main.yml#L3-L7).